x-pack/plugins/stack_connectors/public/connector_types/slack_api/slack_connectors.tsx

```diff
@@ -33,15 +42,123 @@ const getSecretsFormSchema = (docLinks: DocLinksStart): SecretsFieldSchema[] =>
   },
 ];
 
+const getConfigFormSchemaAfterSecrets = (
+  options: EuiComboBoxOptionOption[],
+  isLoading: boolean,
+  isDisabled: boolean
+): ConfigFieldSchema[] => [
+  {
+    id: 'allowedChannels',
+    isRequired: false,
+    label: i18n.ALLOWED_CHANNELS,
+    helpText: (
```

Comment:
Maybe we can add a message explaining why the combo box is disabled if the user has not filled out the secrets.